components/inputs/inputs.css

```diff
@@ -0,0 +1,188 @@
+/**
+ * Copyright 2016 The AMPStart Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.ampstart-input {
+  max-width: 100%;
+  width: 300px;
+  min-width: 100px;
+  font-size: 1rem;
+  line-height: var(--line-height-3);
+}
+
+.ampstart-input > textarea,
+.ampstart-input > select,
+.ampstart-input > input {
+  width: 100%;
+  margin-top: var(--line-height-1);
+  line-height: var(--line-height-3);
+  /* Doing this with border-none class does not set it to none*/
+  border: 0;
+  border-radius: 0;
+  border-bottom: var(--border-width) solid var(--color-primary);
+  background: none;
+  color: var(--color-dark-grey);
+  outline: 0;
+}
+
+.ampstart-input > label {
+  color: var(--color-grey);
+  pointer-events: none;
+  font-size: var(--h5);
+  line-height: var(--line-height-1);
+  opacity: 0;
+  animation: .2s;
+  animation-timing-function: cubic-bezier(.4,0,.2,1);
+  animation-fill-mode: forwards;
+}
+
+.ampstart-input > textarea:focus,
+.ampstart-input > select:focus,
+.ampstart-input > input:focus {
+  outline: 0;
+}
+
+
+/* Move placeholder above */
+.ampstart-input > textarea:focus::placeholder,
+.ampstart-input > select:focus::placeholder,
+.ampstart-input > input:focus::placeholder {
+  color: transparent;
+}
+
+.ampstart-input > textarea:not(:placeholder-shown) + label,
+.ampstart-input > select:not(:placeholder-shown) + label,
+.ampstart-input > input:not(:placeholder-shown) + label {
+  opacity: 1;
+}
+
+.ampstart-input > textarea:focus + label,
+.ampstart-input > select:focus + label,
+.ampstart-input > input:focus + label {
+  animation-name: ampstart-move-input-label;
+}
+
+@keyframes ampstart-move-input-label {
+  to {
+    opacity: 1;
+  }
+}
+
+/* Bottom animation */
+.ampstart-input > label::after {
+  content: '';
+  height: 2px;
+  position: absolute;
+  bottom: 0;
+  left: 45%;
+  background: var(--color-primary);
+  transition: .2s;
+  transition-timing-function: cubic-bezier(.4,0,.2,1);
+  visibility: hidden;
+  width: 10px;
+}
+
+.ampstart-input > textarea:focus + label::after,
```

Comment:
yeah, these are copy paste-able components. Also there is no backward compatibility (atleast for the near future), since this is copy paste and be done with it.